source/parser.hera

```diff
@@ -5918,8 +5918,9 @@ TypeParameterDelimiter
   &( __ ">" )
   &EOS InsertComma -> $2
 
+# TypeScript's this: T syntax in function parameters
```

Comment:
Not sure why 919c680fc5bf6c507a6df29a8d80fe2aca1cbb44 is appearing here again; isn't it already merged into `main`?

Reply:
I branched off a branch by mistake but its should be merged now.